src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/handler/processorder/ProcessOrderSubmittedHandlerTest.java

```diff
@@ -1,11 +1,13 @@
-package uk.gov.hmcts.reform.finrem.caseorchestration.handler;
+package uk.gov.hmcts.reform.finrem.caseorchestration.handler.processorder;
```

Comment:
These tests look good.  

Any value in testing a PROCESS_ORDER event to check:
- log is called
- sendAdditionalHearingDocuments and sendInitialHearingCorrespondence are never called?